front/src/frontend/view/Main.tsx

```diff
@@ -0,0 +1,122 @@
+import 'bootstrap/dist/css/bootstrap.css';
+import '../css/Main.css';
+import React from "react";
+import {Formik, FormikProps} from "formik";
+import axios from "axios";
+
+class RegisterDTO {
+    constructor(
+        public nickname: string,
+        public email: string,
+        public sex: string,
+        public traits: TraitsDTO
+    ) {
+    }
+}
+
+class TraitsDTO {
+    constructor(public extroversion: number,
+                public neuroticism: number,
+                public agreeableness: number,
+                public conscientiousness: number,
+                public openness_to_experience: number
+    ) {
+    }
+}
+
+
+export class Main
+    extends React.Component<{}, {}> {
+
+    bigFive = ["extroversion", "neuroticism", "agreeableness", "conscientiousness", "openness_to_experience"];
```

Comment:
Try to remove dependency by extracting it from class fields. (Is it even possible?)